contracts/token/ERC1155/extensions/ERC1155URIStorage.sol

```diff
@@ -17,7 +17,7 @@ abstract contract ERC1155URIStorage is ERC1155 {
     string private _baseURI = "";
 
     // Optional mapping for token URIs
-    mapping(uint256 => string) private _tokenURIs;
+    mapping(uint256 tokenId => string uri) private _tokenURIs;
```

Comment:
Its not really the uri, its the "uriExtension" that get concatenated with the `_baseURI`. Not sure if we want to reflect that.

Reply:
I'd leave it as it is right now because I see we're also calling `tokenUri` to the variable itself and even when we concatenate with the `baseUri`. I agree with renaming them all but will prefer doing in another PR.